Mono.Debugging.Soft/PortablePdbData.cs

```diff
@@ -153,8 +113,8 @@ SourceLinkMap[] GetSourceLinkMaps ()
 
 		internal SoftScope [] GetHoistedScopesPrivate (MethodMirror method)
 		{
-			using (var fs = new FileStream (pdbFileName, FileMode.Open, FileAccess.Read, FileShare.Read))
-			using (var metadataReader = MetadataReaderProvider.FromPortablePdbStream (fs)) {
+			using (var stream = GetStream())
+			using (var metadataReader = MetadataReaderProvider.FromPortablePdbStream (stream)) {
```

Comment:
Not necessary, but would be good to do. Instead of reading the bytes, we could get the blobreader and read integer by integer.

Read a compressed integer for the length, and then read 2 integer values for the values we're looking for.

Reply:
This was existing code

Reply:
Gotcha. Maybe in a future iteration. If you want, I can file a VSTS ticket

Reply:
Yeah, please

Reply:
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/960770